app/src/test/java/com/android/streetworkapp/model/park/ParkViewModelTest.kt

```diff
@@ -192,7 +192,7 @@ class ParkViewModelTest {
     parkViewModel.getParkByPid("123")
     testDispatcher.scheduler.advanceUntilIdle()
 
-    val observedPark = parkViewModel.park.first()
+    val observedPark = parkViewModel.currentPark.first()
```

Comment:
Question: same confusion problem is there a difference between a currentPark and a park? If so, when is it useful?